Formula/a/autopep8.rb

```diff
@@ -17,15 +17,24 @@ class Autopep8 < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "bd2c70a4d66684833df256b1c1ede1240b09c9c89e434ead620840ed0c3d332f"
   end
 
+  depends_on "pycodestyle"
```

Comment:
pycodestyle is a single script so we only install into `bin` and not site packages: https://github.com/Homebrew/homebrew-core/blob/e511347773cdf57482af09696f33d8e07daccd5b/Formula/p/pycodestyle.rb#L25

changing the path in the `.pth` file won't work either since it doesnt have a `.py` file extension, so don't think we can depend on it

Reply:
👍